index.html

```diff
@@ -35,7 +35,7 @@ <h1>I'm Mustafa. <br>
             <li class="peace-icon"><a href=""><img src="Icons/peace.png" alt="peace"></a></li>
             </ul>
     </section>
```

Comment:
- [ ] [ OPTIONAL ] This change is optional because it does not belong to this milestone, yet I am advising you to implement it to enhance your design 😁 😁 . Currently, you have too much vertical spacing at the end of this section (look at the screenshot below 👇🏽 👇🏽 

![Screenshot from 2023-06-03 09-24-50](https://github.com/MustafaSamiShirzad/Portifoliowebsite/assets/10439283/9c8a09bd-6410-4352-bbf1-ef4b2c97c054)

You can fix it by setting <code>height: auto;</code> in your <code>headline-container</code> class 😄